.github/workflows/package.yml

```
@@ -0,0 +1,40 @@
+name: Package
+
+on:
+  push:
+      branches: [ "develop"]
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)